carbonmark-api/src/utils/helpers/utils.ts

```diff
@@ -228,3 +228,14 @@ export const isMatchingCmsProject = (
   { registry, projectId }: IsMatchingCmsProjectArgs,
   project: CarbonProject
 ) => project?.registryProjectId === projectId && project.registry === registry;
+
+export function formatGraphTimestamps<
+  T extends { createdAt: string | null; updatedAt: string | null },
+>(data: T) {
+  const partialData = omit(data, ["createdAt", "updatedAt"]);
+  return {
+    ...partialData,
+    createdAt: Number(data.createdAt),
```

Comment:
Are the graph time stamps not in milliseconds? i.e we don't need to divide by 1000

If so can you put a short comment in to mention this?

```suggestion
/** Note: Graph timestamps are in seconds **
    createdAt: Number(data.createdAt),
```